docs/rest_api.rst

```diff
@@ -207,10 +207,66 @@ These endpoints allow mark courses as a public.
 This means the course is available to
 all site learners regardless of their Course Access Groups membership.
 
+Public Course Marks
+~~~~~~~~~~~~~~~~~~~
+
+This endpoint returns a paginated list of JSON objects in "results".
+Each object represents a public course mark.
```

Comment:
@OmarIthawi What specifically is a "course mark"? Is this like a grade?

Reply:
Good catch. I refactored this to make it more obvious.